aten/src/ATen/native/mkldnn/xpu/detail/QConv.cpp

```diff
@@ -48,29 +49,29 @@ qconv_get_md(
 }
 
 at::Tensor quantized_convolution(
-    at::Tensor act,
+    const at::Tensor& act,
     double act_scale,
     int64_t act_zero_point,
-    at::Tensor weight,
+    const at::Tensor& weight,
     at::Tensor weight_scales,
-    at::Tensor weight_zero_points,
+    const at::Tensor& weight_zero_points,
     std::optional<at::Tensor> bias,
-    torch::List<int64_t> stride,
-    torch::List<int64_t> padding,
-    torch::List<int64_t> dilation,
+    const torch::List<int64_t>& stride,
+    const torch::List<int64_t>& padding,
+    const torch::List<int64_t>& dilation,
     bool transposed,
     int64_t groups,
     at::Tensor output,
     double inv_output_scale,
     int64_t output_zero_point,
-    std::optional<at::Tensor> accum,
+    const std::optional<at::Tensor>& accum,
     double accum_scale,
     int64_t accum_zero_point,
     std::optional<c10::ScalarType> output_dtype,
     std::optional<std::string_view> binary_attr,
     std::optional<at::Scalar> binary_alpha,
     std::optional<std::string_view> unary_attr,
-    torch::List<std::optional<at::Scalar>> unary_scalars,
+    const torch::List<std::optional<at::Scalar>>& unary_scalars,
     std::optional<std::string_view> unary_algorithm) {
```

Comment:
Why other inputs like the act_scale, act_zero_point and weight_scales can't be const?

Reply:
There can be, I will add more const